docs/ecu-ElmProtocolInit.js

```
@@ -0,0 +1,1351 @@
+            {
+              address,
+              command: cmd,
+              response: cmdResponse,
+            },
+          );
+        }
+
+        await this.delay(DELAYS.PROTOCOL);
+      }
+
+      const testCmd = SINGLE_TEST_COMMAND;
```

ESLint

[error] 'SINGLE_TEST_COMMAND' is not defined.

(no-undef)